### PR TITLE
Implement adding user organization

### PR DESCRIPTION
**Pull request by scoiatael**
Fixes https://github.com/scoiatael/zosia16-site/issues/85

### CODE REVIEW

File: conferences/forms.py [hunk 1]
```diff
@@ -17,7 +18,20 @@ def __init__(self, *args, **kwargs):
...
+class UserPreferencesWithOrgForm(UserPreferencesWithBusForm):
+    def org_queryset(self, user):
+        org_queryset = Organization.objects.filter(accepted=True)
+        org_queryset = org_queryset | Organization.objects.filter(user=user)
+        return org_queryset
+
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        org_form = self.fields['organization']
+        org_form.widget = SelectWithAjaxAdd()
```
Comment (szewczykmira) [inline conferences/forms.py]: you can define widget in `Meta` class

File: conferences/widgets.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            value = ''
+        attrs['class'] = attrs.get('class', '') + ' col s11'
+        final_attrs = self.build_attrs(attrs, name=name)
+        # TODO: generate new id
+        output = ['<div class="row">', format_html('<select id="org-select" {}>', flatatt(final_attrs))]
+        options = self.render_options([value])
+        if options:
+            output.append(options)
+        output.append('</select>')
+        # TODO: trans
```
Comment (szewczykmira) [inline conferences/widgets.py]: why so many todo?

File: users/models.py [hunk 3]
```diff
@@ -21,6 +21,17 @@ class Organization(models.Model):
     accepted = models.BooleanField(
         default=False
     )
+    user = models.ForeignKey(
+        User,
+        null=True,
+        blank=True)
+
+    @property
```
Comment (szewczykmira) [inline users/models.py:29]: why do you need owner?

Comment (scoiatael) [reply users/models.py:29]: `__str__` helper ;)

File: conferences/widgets.py (hunk 2 above)
Comment (scoiatael) [reply conferences/widgets.py]: Not to forget?

Comment (szewczykmira) [reply conferences/widgets.py]: Then maybe do it instead of creating todo? ;)

File: users/views.py [hunk 4]
```diff
@@ -72,3 +74,14 @@ def activate(request, uidb64, token):
         'user': action.user,
     }
     return render(request, 'users/activate.html', ctx)
+
+
+@login_required
+@require_http_methods(['POST'])
+def create_organization(request):
+    user = request.user
+    name = request.POST.get('name', None)
```
Comment (szewczykmira) [inline users/views.py:83]: FYI - by default if `get` doesn't find result it will return `None` :)

**Review by szewczykmira — APPROVED** [review]

**Pull request merged**